src/components/Header.js

```diff
@@ -89,25 +20,9 @@ const Hero = styled.div`
 const Header = () => (
     <StyledHeader role='banner' id="top">
         <div className='row'>
-            <div className="svg" aria-hidden={true}>
-                <div className="container">
-                    <img
-                        src={Stream}
-                        className='stream'
-                        alt='Plance Stream'
-                    />
-                    <img
-                        src={Plane}
-                        className='plane'
-                        alt='Paper Plane'
-                    />
-                    &nbsp;
-                </div>
-                &nbsp;
-            </div>
             <Hero>
-                <h1>Frictionless Open-Source Contributions</h1>
-                <p className="seconed">Setting up a local dev environment can be daunting. - All the projects below have automated their dev setup with <a href="https://www.gitpod.io" target="_blank">Gitpod.io</a>, a free online IDE for GitHub. <span style={{ fontWeight: 600 }}>Now your first Open-Source contribution is just one click away.</span></p>
+                <h1><strong>Frictionless</strong> Open-Source Contributions</h1>
+                <p>All projects below have automated their dev setup with <a href="https://www.gitpod.io" target="_blank">Gitpod</a>.</p>
```

Comment:
This needs to be explained a bit more. What about:
"Setting up a local dev environment can be daunting. All projects below describe their dev environment as code with Gitpod. `<br>` Now you can start contributing with a single click."

Reply:
Thanks. This is how it looks with this change in place. I feel like the width of the paragraph is a bit long:

![image](https://user-images.githubusercontent.com/46004116/87538823-51fb6980-c6b6-11ea-8652-3771ece2ab2d.png)

reducing the width with the `<br />` in place does not look good either:

![image](https://user-images.githubusercontent.com/46004116/87538936-7a836380-c6b6-11ea-817e-8ef6ab4d5ccc.png)

Would it be ok if I were to get rid of the `<br />`? to me it looks better without the `<br />`:

![image](https://user-images.githubusercontent.com/46004116/87539050-ad2d5c00-c6b6-11ea-886e-18d4bd7d72e3.png)

Reply:
Jep, fine by me  👍